react/tic-tac-toe/src/app/screens/Matches/index.js

```diff
@@ -1,23 +1,25 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import Spinner from 'react-spinkit';
+import PropTypes from 'prop-types';
 
 import MatchesService from '../../../services/MatchesService';
+import actionsCreators from '../../../redux/Matches/actions';
 
 import styles from './styles.module.scss';
 
-class Matches extends Component {
-  state = {
-    data: []
-  }
 
+class Matches extends Component {
   componentDidMount() {
-    MatchesService.getMatches().then(({ data }) => {
-      this.setState({ data });
-    });
+    this.props.dispatch(this.loadData());
   }
 
+  loadData = () => dispatch => MatchesService.getMatches().then(({ data }) => {
```

Comment:
A couple of thing here:
* As a convention we follow, we rename `actionCreators` to something more descriptive
```js
import actionCreators as matchesActions from '../../../redux/Matches/actions';
```
* Services calls should be encapsulated under redux actions scope and using redux-thunk
* Actions calls should be defined inside `mapDispatchToProps` and used as regular props.
```js
// Component
...
componentDidMout() {
   this.props.getMatches();
}
...

// similar to mapStateToProps
const mapDispatchToProps = dispatch => ({
   getMatches: () => dispatch(...)
});
```